app/src/main/java/com/nervousfish/nervousfish/modules/constants/Constants.java

```diff
@@ -37,6 +39,10 @@
      */
     private Constants(final IServiceLocator serviceLocator) {
         this.androidFilesDir = serviceLocator.getAndroidFilesDir();
+        //noinspection StringConcatenationMissingWhitespace because this is a file path
+        this.databaseContactsPath = this.androidFilesDir + Constants.DB_CONTACTS_PATH;
+        //noinspection StringConcatenationMissingWhitespace because this is a file path
+        this.databaseUserPath = this.androidFilesDir + Constants.DB_USERDATA_PATH;
```

Comment:
I did it, because it's inefficient to concatentate Strings every single time you call the getter. You only want to do that once.